### PR TITLE
Add styling for bib HTML and LaTeX output

### PR DESCRIPTION
**Pull request by adelhult**
Resolves GH-323. This PR adds enhanced 2 column styling for the bibliography when using the HTML or the LaTeX output format.
**Also, a bit unrelated:**  I ended up playing around with using a imports variable in the html template so this is included in the PR too.

**HTML**
Screenshot from 2023-05-13 07-42-51

**LaTeX**
Screenshot from 2023-05-12 22-20-02

**Plain text fallback**
Screenshot from 2023-05-12 22-19-07

### CODE REVIEW

File: packages/bibliography/src/main.rs [hunk 1]
```diff
@@ -285,39 +289,133 @@ fn transform_bibliography(to: &str, input: &Value) {
...
+    match to {
+        // We want to use a tabularx environment when outputing latex where the prefix
+        // is in the first column and the rest in the second
+        "latex" => {
+            output.push(raw!("\n"));
+            output.push(import!("\\usepackage{tabularx}"));
+
+            output.push(raw!(
+                r"\renewcommand{\arraystretch}{1.5}
+\begin{tabularx}{\textwidth}{p{0.3cm} X}
```
Comment (Forsinge) [inline packages/bibliography/src/main.rs:340]:
```suggestion
\begin{tabularx}{\textwidth}{p{0.3cm} >{\raggedright\arraybackslash}X}
```

**Review by Forsinge — COMMENTED** [review]
Nice work! Would be nice to also avoid the horizontal stretching in each entry. I commented one way to achieve that.

**Comment by adelhult** [comment]
Most of the issues pointed out should now be fixed. I only use a single column (or in the case of HTML a flexbox layout instead) when there is no prefix. The module can also handle corner cases where some entries have a prefix while others do not.
Screenshot from 2023-05-13 15-16-27

Screenshot from 2023-05-13 15-17-50

File: packages/bibliography/src/main.rs [hunk 2]
```diff
@@ -286,38 +290,204 @@ fn transform_bibliography(to: &str, input: &Value) {
...
+            );
+
+            output.push(raw!(
+                r"\end{tabularx}
+\renewcommand{\arraystretch}{1}
+"
+            ));
+            output
+        }
+        // For html output we use a simple css grid layout
```
Comment (Forsinge) [inline packages/bibliography/src/main.rs:403]: It looks like bibliographies without prefix use `display: flex` now, could update comment to reflect that

**Review by Forsinge — APPROVED** [review]
Seems like it works well now, good job!

File: packages/bibliography/src/main.rs (hunk 2 above)
Comment (adelhult) [reply packages/bibliography/src/main.rs:403]: thanks! Will fix!

**Pull request merged**